hikari/guilds.py

```diff
@@ -486,26 +486,20 @@ def is_system(self) -> bool:
     def mention(self) -> str:
         """Return a raw mention string for the given member.
 
-        If the member has a known nickname, we always return
-        a bang ("`!`") before the ID part of the mention string. This
-        mimics the behaviour Discord clients tend to provide.
-
         Example
         -------
 
         ```py
-        >>> some_member_without_nickname.mention
+        >>> some_member.mention
         '<@123456789123456789>'
-        >>> some_member_with_nickname.mention
-        '<@!123456789123456789>'
         ```
 
         Returns
         -------
         builtins.str
             The mention string to use.
         """
-        return f"<@!{self.id}>" if self.nickname is not None else self.user.mention
+        return self.user.mention
```

Comment:
Member inherits from User, so this property could be removed all together from here and it will just fallback to User.mention

Reply:
I thought so too, however I thought it would be more consistent with the other properties: https://github.com/hikari-py/hikari/blob/b56b79c0a2c4ccfa10dab51ff169cb926c518307/hikari/guilds.py#L404-L549

Reply:
Oh, nvm, just remembered why we do it this way. You can keep it, maybe the docstring is what can go only